### PR TITLE
added-app

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [X ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/armis.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+id: armis
+title: Sumo Logic App for Armis
+sidebar_label: Armis
+description: Sumo Logic App for Armis offers complete device security with alert analysis, device monitoring, and device information for IT infrastructure protection.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/armis.md:5]:
```suggestion
description: The Sumo Logic App for Armis offers complete device security with alert analysis, device monitoring, and device information for IT infrastructure protection.
```

File: docs/integrations/saas-cloud/armis.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+```sql title="Device"
+_sourceCategory=ArmisDashboards id
+| json "id","name","manufacturer","model","riskLevel","sensor","site.name","type","category","operatingSystem" as id, name, manufacturer, model, riskLevel, sensor, site, type, category, operatingSystem nodrop
+| where site matches "{{Site}}" and manufacturer matches "{{Manufacturer}}" and type matches "{{Type}}"
+| count_distinct(id)
+```
+
+## Collecting Logs for Armis Api
+
+This section explains how to collect logs from Armis Api and ingest them into Sumo Logic. Refer to the [Armis Api Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source/) to create the source and use the same source category while installing the app.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/armis.md:192]:
```suggestion
## Collecting Logs for Armis API

This section explains how to collect logs from Armis API and ingest them into Sumo Logic. Refer to the [Armis API Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source/) to create the source and use the same source category while installing the app.
```

File: docs/integrations/saas-cloud/armis.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+   3. **Advanced**. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+4. Click **Add to Library**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing the Armis Dashboards​
+
+* All dashboard have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/armis.md:219]:
```suggestion
* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
```

File: docs/integrations/saas-cloud/armis.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+ You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+
+* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+
+
+## Alerts Overview Dashboard
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/armis.md:226]:
```suggestion
Subsection of "Viewing Dashboards"
### Alerts Overview Dashboard
```

File: docs/integrations/saas-cloud/armis.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+**Armis - Alerts Overview dashboard**. The dashboard provides a comprehensive analysis of alerts, by categorizing them based on their severity, status, type, and presenting a trend of alerts over a specified period of time.<br/><img src={useBaseUrl('img/integrations/saas-cloud/Armis-Alerts-Overview.png')} alt="Armis-Alerts-Overview.png" width="600"/>
+
+
+## Device Overview Dashboard
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/armis.md:231]:
```suggestion

Subsection of "Viewing Dashboards"

### Device Overview Dashboard
```

File: docs/integrations/saas-cloud/armis.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+**Armis - Device Overview dashboard**. The dashboard provides a table view of last-seen devices with key information to protect your IT infrastructure and offers visibility into devices based on risk levels, types, and categories.<br/><img src={useBaseUrl('img/integrations/saas-cloud/Armis-Device-Overview.png')} alt="Armis-Alerts-Overview.png" width="600"/>
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/armis.md:233]:
```suggestion
**Armis - Device Overview** dashboard. The dashboard provides a table view of last-seen devices with key information to protect your IT infrastructure and offers visibility into devices based on risk levels, types, and categories.<br/><img src={useBaseUrl('img/integrations/saas-cloud/Armis-Device-Overview.png')} alt="Armis-Alerts-Overview.png" width="600"/>
```

File: docs/integrations/saas-cloud/armis.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+* All dashboard have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
+
+ You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+
+* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+
+
+## Alerts Overview Dashboard
+
+**Armis - Alerts Overview dashboard**. The dashboard provides a comprehensive analysis of alerts, by categorizing them based on their severity, status, type, and presenting a trend of alerts over a specified period of time.<br/><img src={useBaseUrl('img/integrations/saas-cloud/Armis-Alerts-Overview.png')} alt="Armis-Alerts-Overview.png" width="600"/>
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/armis.md:228]:
```suggestion
**Armis - Alerts Overview** dashboard. The dashboard provides a comprehensive analysis of alerts, by categorizing them based on their severity, status, type, and presenting a trend of alerts over a specified period of time.<br/><img src={useBaseUrl('img/integrations/saas-cloud/Armis-Alerts-Overview.png')} alt="Armis-Alerts-Overview.png" width="600"/>
```

**Review by kimsauce — DISMISSED** [review]
Formatting edits

File: docs/integrations/saas-cloud/armis.md (hunk 4 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/armis.md:226]: > 
Thanks, kim for the suggestion. I belive we don't need to mention that this is a subsection as i have used the third heading.

File: docs/integrations/saas-cloud/armis.md (hunk 5 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/armis.md:231]: Thanks, kim for the suggestion. I belive we don't need to mention that this is a subsection as i have used the third heading.

**Comment by JainM6** [comment]
> Formatting edits

Resolved all, except the one suggestion that says "Subsection..."

File: docs/integrations/saas-cloud/armis.md (hunk 4 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/armis.md:226]: The heading you've used is the second (H2) heading (`## Alerts Overview Dashboard`). Should be third (H3).

File: docs/integrations/saas-cloud/armis.md (hunk 5 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/armis.md:231]: The heading you've used is the second (H2) heading. Should be third (H3).

File: docs/integrations/saas-cloud/armis.md (hunk 4 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/armis.md]: Yeas that's correct. Thank you for pointing out this one.

**Review by kimsauce — APPROVED** [review]

File: docs/integrations/saas-cloud/armis.md (hunk 5 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/armis.md]: I see the change now - just approved!

File: docs/integrations/saas-cloud/armis.md (hunk 4 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/armis.md]: I see the change now - just approved!

**Pull request merged**